content/200-concepts/100-components/01-prisma-schema/06-relations/200-one-to-many-relations.mdx

```diff
@@ -205,42 +205,42 @@ In MongoDB, the only difference between a 1-1 and a 1-n is the number of documen
 A 1-n-relation always has two relation fields:
 
 - a [list](/concepts/components/prisma-schema/data-model#type-modifiers) relation field which is _not_ annotated with `@relation`
-- the [annotated relation field](/concepts/components/prisma-schema/relations#relation-fields) (including its relation scalar)
+- the [annotated relation field](/concepts/components/prisma-schema/relations#annotated-relation-fields-and-relation-scalar-fields) (including its relation scalar)
 
-The list side of a 1-n relation is **always mandatory**. On the other side of the relation, the annotated relation field and relation scalar can either _both_ be optional, or _both_ be mandatory.
+The annotated relation field and relation scalar of a 1-n relation can either _both_ be optional, or _both_ be mandatory. On the other side of the relation, the list is **always mandatory**.
 
 In the following example, you can create a `Post` without assigning a `User`:
 
 <TabbedContent tabs={[<FileWithIcon text="Relational databases" icon="database"/>, <FileWithIcon text="MongoDB" icon="database"/>]}>
 <tab>
 
 ```prisma
-model User {
-  id    Int    @id @default(autoincrement())
-  posts Post[]
-}
-
 model Post {
   id       Int   @id @default(autoincrement())
   author   User? @relation(fields: [authorId], references: [id])
   authorId Int?
 }
+
+model User {
```

Comment:
I'm not sure if I agree with this shuffling. We're describing 1-n relationships, but if you read this top to bottom this is n-1. I can understand why the description above was changed but IMO these should stay as they were. Open to being wrong.

Reply:
Fair.

I assume July Jan's focus was to put the model with `@relation` first (forward relation), and then the back relation.

Reply:
I don't have a strong opinion here but I don't think changing it makes it clearer necessarily. In (almost) all other code snippets I can think of we write `User` first and then `Post`, so I would keep the order here for consistency.